commands/wait.js

```diff
@@ -66,7 +65,11 @@ async function run (context, heroku) {
 
       cli.action.status(status.message)
 
-      await wait(interval * 1000)
+      await new Promise((resolve, reject) => {
+        setTimeout(() => {
+          resolve()
+        }, interval * 1000)
+      })
```

Comment:
No idea if this is the right way to do it, but it looks like `co-wait` is not the right way to do it--it seems to ignore the wait entirely.